staging/src/k8s.io/apimachinery/pkg/util/managedfields/fieldmanager_test.go

```diff
@@ -393,15 +393,6 @@ func TestNoOpChanges(t *testing.T) {
 	if !reflect.DeepEqual(before, f.Live()) {
 		t.Fatalf("No-op update has changed the object:\n%v\n---\n%v", before, f.Live())
 	}
-	before = f.Live()
```

Comment:
This test is equivalent to [`TestNoopChangeCreationTime`](https://github.com/kubernetes/kubernetes/blob/master/test/integration/apiserver/apply/apply_test.go#L253-L300)